lib/mix/tasks/phoenix.new.ex

```diff
@@ -33,6 +34,12 @@ defmodule Mix.Tasks.Phoenix.New do
 
     copy_from template_dir, path, application_name, &EEx.eval_file(&1, binding)
     copy_from static_dir, Path.join(path, "priv/static"), application_name, &File.read!(&1)
+
+    # TODO decide to auto npm install or not
+    if npm_path && Mix.env == :dev do
+      IO.puts "Installing brunch.io dependencies..."
+      System.cmd("npm", ["install", "--prefix", path])
```

Comment:
Should we output all the command to stdout in case there are failures using :into?